src/pages/api/auth/profilePhoto.ts

```
@@ -0,0 +1,56 @@
+      throw new Error(
+        `프로필 사진을 가져오는데 실패했습니다: ${response.statusText}`,
+      );
+    }
+
+    const data = await response.json();
+
+    if (data.photos && data.photos[0]?.url) {
+      return res.status(200).json({ photoUrl: data.photos[0].url });
+    }
+
+    return res.status(404).json({ error: "프로필 사진을 찾을 수 없습니다." });
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)